components/cap-jurisdictions.js

```diff
@@ -0,0 +1,45 @@
+import { LitElement, html, css } from "../lib/lit.js";
+import { fetchJurisdictionsData } from "../lib/data.js";
+
+export default class CapJurisdictions extends LitElement {
+	static properties = {
+		jurisdictionsData: { attribute: false },
+	};
+
+	constructor() {
+		super();
+		this.jurisdictionsData = [];
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		fetchJurisdictionsData(this);
+	}
+
+	render() {
+		return html`
+			<ul>
```

Comment:
If I understand correctly, what this page currently outputs is a list with nested lists:

![Screenshot 2024-01-09 at 2 36 17 PM](https://github.com/harvard-lil/capstone-static/assets/4039311/397f5c3c-3ba3-47f9-ab76-3c2d23b813e6)

I would recommend that we instead output each section as an individual `article` element, containing an h2 element and a single list. 

Individuals who rely on screenreaders will be able to easily jump between headings using their rotor function (which groups landmarks, such as all h2 elements)